AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -2137,6 +2137,18 @@ - (NSString *)_recursiveDescriptionHelperWithIndent:(NSString *)indent
   return subtree;
 }
 
+#pragma mark - ASLayoutableAsciiArtProtocol
+
+- (NSString *)asciiArtString
+{
+    return [ASLayoutSpec asciiArtStringForChildren:@[] parentName:[self asciiArtName]];
```

Comment:
Just to make sure I understand — this only prints out one box with the node's name in it?  Or does it somehow print the hierarchy?